mod-source-record-manager-server/src/test/java/org/folio/rest/impl/changeManager/ChangeManagerAPITest.java

```diff
@@ -1885,4 +1886,48 @@ public void shouldSetErrorToRecordWithInvalidLeaderLine(TestContext testContext)
     MatcherAssert.assertThat(recordCollection.getRecords().get(0).getErrorRecord().getDescription(), containsString("Error during analyze leader line for determining record type"));
   }
 
+  @Test
+  public void testDeleteChangeManagerJobExecutions_SingleEntity(){
+    RestAssured.given()
+      .spec(spec)
+      .body("{\"ids\":[\"15065ccd-c305-4961-a411-8359e0b5a87b\"]}")
+      .when()
+      .delete(JOB_EXECUTION_PATH)
+      .then()
+      .statusCode(HttpStatus.SC_OK);
+  }
+
+  @Test
+  public void testDeleteChangeManagerJobExecutions_MultipleIds(){
+    RestAssured.given()
+      .spec(spec)
+      .body("{\"ids\":[\"15065ccd-c305-4961-a411-8359e0b5a87b\", \"15065ccd-c305-4961-a411-8359e0b5a87b\"]}")
+      .when()
+      .delete(JOB_EXECUTION_PATH)
+      .then()
+      .statusCode(HttpStatus.SC_OK);
+  }
+
+  @Test @Ignore
```

Comment:
don't forget to delete @Ignore annotation

Reply:
@SerhiiNosko - PR updated as per comments, Thanks 🙂